### ISSUE TITLE
Usersnap Feedback - Copy-edit: we should consider changing this response to 'Both/Either' sinc[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#839]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Copy-edit: we should consider changing this response to 'Both/Either' since some methods, tools, and techniques are either totally in-person or totally online, but 'Both' implies simultaneous f2f and online interaction
[Open #536 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/536)

<a href='https://usersnappublic.s3.amazonaws.com/2019-08-28/20-52/49c875f5-6abb-47f8-a87f-f11f7c64cc3c.png'>![Usersnap Feedback - Copy-edit: we should consider changing this response to 'Both/Either' sinc(...)](https://usersnappublic.s3.amazonaws.com/2019-08-28/20-52/5780e018-bdb4-4667-ab4a-13304a58493e.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-08-28/20-52/49c875f5-6abb-47f8-a87f-f11f7c64cc3c.png'>Download original image</a>

**Browser**: Chrome 76 (Windows 10)
**Referer**: [https://participedia.net/method/new](https://participedia.net/method/new)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 873

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by scottofletcher** [comment]
@ascott how do I change this? can I use PhraseApp?

**Comment by ascott** [comment]
@scottofletcher yes, you can use Phrase app to change the heading, instructional text and values to choose from. The instructions to this are linked in the translators channel if you need them, but basically just search for the english phrase/word you want to change and you'll find the correct key to update. Let me know if you need help. 

As as aside, Both doesn't imply simultaneously to me, it reads as intended, that the method can use both of these interaction methods. I think it's obvious that it wouldn't be simultaneously face-to-face and online.

Another thing to consider with this change, is that the French translations have just been verified, so unless we absolutely have to, I would suggest we don't make copy changes to the form so the translations are all in sync.

**Comment by plscully** [comment]
@scottofletcher I agree with Alanna. I would leave this text as is. I never interpreted 'both' to mean 'simultaneously'

**Comment by scottofletcher** [comment]
Sorry, I didn't mean simultaneously, I meant that selecting 'Both' for a method (or a tool/technique) implies that there are both online and in-person components. for example, if you select 'Both' for participatory budgeting, that, to me, would imply that the process contains online _and_ in-person elements of participation which isn't necessarily the case. some PBs are totally in-person, some are totally online, and some do, indeed, have a mixture of the two. having 'Both/Either' would capture that better (imo).

**Comment by jesicarson** [comment]
If it is totally in person you'd select "face to face." If it is totally online you'd select "online." If it is a mix of any kind, you'd select "both." 
Adding "either" to the "both" option makes the other two options redundant, doesn't it?

**Comment by scottofletcher** [comment]
No because some methods, tools, and techniques can be either. For example, participatory budgeting can be both or it can be either. some municipalities use a purely online process (eg. Reykjavik), some use a purely face-to-face process (eg. [Enschede](https://participedia.net/case/2842)), and some use both (eg. Paris, Madrid, etc.). Ergo, we need a both/either selection for the method entry on Participatory Budgeting

**Comment by scottofletcher** [comment]
to be clear: this is for method and tool/technique entry types, not cases.

**Comment by jesicarson** [comment]
ok cool. you can add this to your forms feedback tracking doc and we'll address it during the next review.

**Comment by scottofletcher** [comment]
Thanks! I'll add an editor tag to this and reference it in the editors repo
